<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreRecordUsersSetupDialog</class>
 <widget class="QDialog" name="PreRecordUsersSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="4" column="1" colspan="4">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="QCheckBox" name="chkDetailCallCenter">
     <property name="text">
      <string>Детализировать Call-центр</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>&amp;Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegRecordDate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtBegScheduleDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="spacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblSpeciality">
     <property name="text">
      <string>&amp;Специальность</string>
     </property>
     <property name="buddy">
      <cstring>cmbSpeciality</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblEndScheduleDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>по</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblEndRecordDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>по</string>
     </property>
     <property name="buddy">
      <cstring>edtEndRecordDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkSchedulePeriod">
     <property name="text">
      <string>Период планируемого приёма с</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbSpeciality">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>&amp;Врач</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="CDateEdit" name="edtEndRecordDate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="CDateEdit" name="edtEndScheduleDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="chkRecordPeriod">
     <property name="text">
      <string>Период постановки в очередь с</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="QCheckBox" name="chkSpecialityFilter">
     <property name="text">
      <string>Врачи-специалисты</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="QCheckBox" name="chkShowDeleted">
     <property name="text">
      <string>Учитывать удаленные записи</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkRecordPeriod</tabstop>
  <tabstop>edtBegRecordDate</tabstop>
  <tabstop>edtEndRecordDate</tabstop>
  <tabstop>chkSchedulePeriod</tabstop>
  <tabstop>edtBegScheduleDate</tabstop>
  <tabstop>edtEndScheduleDate</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbSpeciality</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>chkSpecialityFilter</tabstop>
  <tabstop>chkDetailCallCenter</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkRecordPeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtBegRecordDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkRecordPeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtEndRecordDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSchedulePeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtBegScheduleDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSchedulePeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtEndScheduleDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreRecordUsersSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreRecordUsersSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSchedulePeriod</sender>
   <signal>clicked()</signal>
   <receiver>edtBegScheduleDate</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkRecordPeriod</sender>
   <signal>clicked()</signal>
   <receiver>edtBegRecordDate</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
